### PR TITLE
fix: [CCM-5536]: HDurations fix

### PR DESCRIPTION
**Pull request by trunapushpa**
You can use the following comments to re-trigger PR Checks

- Compile: `trigger compile`
- runAeriformCheck: `trigger AeriformCheck`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`

---
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/30951)

### CODE REVIEW

**Review by hitesharinga — DISMISSED** [review]
> Reviewed 2 of 2 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/30951)* status: :shipit: complete! ✓

File: 970-grpc/src/test/java/io/harness/grpc/utils/HDurationsTest.java [hunk 1]
```diff
@@ -51,4 +51,11 @@ public void shouldParseMinute() throws Exception {
   public void shouldParseMinuteAndSecond() throws Exception {
     assertThat(HDurations.parse("1m23s")).isEqualTo(Durations.fromSeconds(83));
   }
+
+  @Test
+  @Owner(developers = AVMOHAN)
```
Comment (nikunj-badjatya) [inline 970-grpc/src/test/java/io/harness/grpc/utils/HDurationsTest.java]: You can change this to your name probably ?

**Review by nikunj-badjatya — APPROVED** [review]
> Reviewed 2 of 2 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/30951)* status: :shipit: complete! ✓

**Review by ketankr9 — APPROVED** [review]
> Reviewed 2 of 2 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/30951)* status: :shipit: complete! ✓

**Pull request merged**